### ISSUE TITLE
How do you link to gd

### ISSUE THREAD

**Issue by catsareamazing123** [absoIute/Mega-Hack-v5#31]

**Comment by atSquiDevONtwitter** [comment]
<h1><strong>GitHub user &#39;catsareamazing123&#39; opened an issue (#31) in the github directory of:</strong></h1>

<p>&nbsp;</p>

<h1>&lt;DIR&gt;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; /absoIute/Mega-Hack-v5/issues/31/</h1>

<p>&nbsp;</p>

<h2><strong>GitHub user atSquiDevONtwitter responds.</strong></h2>

<p><strong>If you need more help, read the big text below this text.</strong></p>

<p><strong>Now, Open GeometryDash.exe, then wait for it to load, then launch MegaHack.exe. Now keep in mind that you will have to repeat the whole process if any program closes.</strong></p>

<p>&nbsp;</p>

<h1><strong>To start off, make sure both programs are closed. Since I believe you are a Microsoft Windows User, I will tell you how to get the program running. First off, make sure that no important files are corrupted in your GeometryDash directory. Ways they could be corrupted could include the following: Malicious Software; EncryptionWare like .shadow (NOT the same as Windows Shadow Copy. You can possibly recover .shadow by downloading <a href="https://www.shadowexplorer.com/uploads/ShadowExplorer-0.9-portable.zip">this [Click Here]</a> and allowing the program in your antivirus.), or possible missing _CommonRedist .dll files that aren&#39;t accepted for Driver Installation from Steam&#39;s &#39;drivers.exe&#39; file. You can find the _CommonRedist installation files for GeometryDash on my <a href="https://github.com/atSquiDevONtwitter/GeometryDash-_CommonRedist-Install">GitHub Page [Click Here]</a>. [Quick extra note, you can find your GeometryDash folder in \Users\&#39;Your Username&#39;\Steam\steamapps\common\GeometryDash or in your ProgramsFiles or ProgramFiles (x86) folders.) Now with MegaHackV.5. Make sure to allow the *.zip and the extracted folder in your antivirus (*No need to download WinRAR or 7zip. I recommend them ONLY for the extraction of .rar, .rar version 6, and .7z. Windows has normal .zip file fuctionalities built in, no one knows all the functionalities that explorer.exe has.), then navigate to the extracted folder until you see [bin.folder] [src.folder] [.gitattributes] and [README.md]. all we need to open is [bin.folder], then Right-Click [MegaHack.exe], hover over [Send to] then click [Desktop (Create Shortcut)]. Now, Open GeometryDash.exe, then wait for it to load, then launch MegaHack.exe. Now keep in mind that you will have to repeat the whole process if any program closes.</strong></h1>

**Comment by catsareamazing123** [comment]
Thanks but Can you link it to mobile?

**Comment by sha5672** [comment]
No

**Comment by multiwastaken** [comment]
no, get italian apk downloader's mod menu for android if you want a mobile replacement
